src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,136 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.exception.InvalidInputException;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static final String CORRECT_LOGIN = "login555";
+    private static final String CORRECT_PASSWORD = "password";
+    private static final int CORRECT_AGE = 20;
+    private static User user;
+    private static User userTwo;
+    private static RegistrationService registrationService;
+
+    @BeforeAll
+    static void beforeAll() {
```

Comment:
Best practice is to initialise data that you are going to test in each test method -> it's increase readability of test. If you have 30 test cases developer don't need to see at the start of Test class each time to refresh memory what exactly you testing